sandboxes/dynamicanalysis/Dockerfile

```diff
@@ -63,6 +63,9 @@ RUN apt-get update && apt-get -y upgrade && apt-get install -y --no-install-reco
 # Configure sudo for passwordless execution
 RUN echo "ALL ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 
+# Create ssh directory for root
+RUN mkdir /root/.ssh
```

Comment:
I you may also want to add a `&& chmod 0750 /root/.ssh` to the end to ensure the perms are somewhat realistic.

Reply:
Sure, will do